scanpy/tests/test_highly_variable_genes.py

```diff
@@ -85,7 +85,7 @@ def test_no_batch_matches_batch(adata):
 
 
 @pytest.mark.parametrize("batch_key", [None, "batch"], ids=["single", "batched"])
-@pytest.mark.parametrize("array_type", ARRAY_TYPES_SUPPORTED)
+@pytest.mark.parametrize("array_type", ARRAY_TYPES)
```

Comment:
Does the fact this works means we support dask sparse chunks with hvg or is there something else we need to test?  Should update release notes/test that thing if so.